client/views/incidentReports/incidentTable.jade

```diff
@@ -9,26 +9,20 @@ template(name="incidentTable")
         th Start Date
         th End Date
         th.locations-header Locations
-        th Status
```

Comment:
It may be a good idea to do so. That way we can avoid the logic involved in not showing multi-select and related UI elements.